lib/workerholic/job.rb

```diff
@@ -40,7 +40,7 @@ def verify_delay(delay_arg)
     def prepare_job_for_enqueueing(args)
       raise ArgumentError if self.method(:perform).arity != args.size
 
-      job = JobWrapper.new(class: self.class, arguments: args)
+      job = JobWrapper.new(klass: self.class, arguments: args)
```

Comment:
Reserved keyword (highlighted by the syntax checker). It might be OK as a hash key, but I thought it might be better to replace it with what is considered as good practice. This way there will be no confusion about the meaning.